src/stores/notification.ts

```diff
@@ -76,22 +114,60 @@ function addId(notification: NotificationItem): NotificationItem {
   };
 }
 
+function mapApiResponseMessage(message: ApiResponseMessage, type: 'success' | 'info' | 'error'): NotificationItem {
+  const { secondsToShow } = message || {};
+
+  return {
+    type,
+    message: message.message,
+    expire: typeof secondsToShow === 'number' && secondsToShow > 0,
+    pageReload: message.action === ApiResponseMessageAction.PageReload,
+    redirectUrl: message.redirectUrl || undefined,
+  };
+}
+
+export function mapApiResponseMessages(messages: ApiResponseMessages): NotificationItem[] {
+  const { SUCCESS, INFO, ERROR } = messages || {};
```

Comment:
question: uppercase keys seems pretty unconventional. Can we at least make them camelCase in our template?

Reply:
Some devs prefer uppercase in recent projects, but we can choose camelCase if we prefer in our template. As a team, we should probably agree on a convention, or not.

Reply:
I changed it for the template now.